tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.4.2",
+    "compilerOptions": {
+        "lib": [
+          "es2017",
+          "esnext.asynciterable"
+        ],
+        "target": "es2017",
+        "module": "commonjs",
+        "moduleResolution": "node",
+        "emitDecoratorMetadata": true,
+        "experimentalDecorators": true,
+        "declaration": true,
+        "sourceMap": true,
+        "strict": true,
+        "noImplicitAny": true,
```

Comment:
Part of strict

Reply:
I checked and you're right!  But it's so confusing, I reckon I'll keep it anyway.